### DISCUSSION TITLE
Can't install motionplus app on linux mint 21.3

### DISCUSSION THREAD

**Discussion by adamromek** [Motion-Project/motionplus#179] [General]
Motion 4.6 works, but I can't install MotionPlus0.2.0.
The libcamera0.0.3 libraries are missing.
The built build.sh file doesn't work in Linux Mint. It says Debian, Ubuntu and doesn't want to work for me.
I would like to have audio recorded as well because motion itself doesn't have this option.

Can someone tell me where the problem is?
Best regards, Adam Romek

**Comment by Mr-Dave** [comment]
> The built build.sh file doesn't work in Linux Mint. It says Debian, Ubuntu and doesn't want to work for me.

Are you referring to the [building your own deb package script](https://motion-project.github.io/motionplus_build.html#Build_Deb) buildplus.sh ?

If so, you may be able to hack it by just specifying `DISTO=Ubuntu` right after the `#  0.  Validate d....` documentation line.  Otherwise, you'd need to [build from source](https://motion-project.github.io/motionplus_build.html#Building) or go into more depth debugging the script on what is different(version numbers, packages, etc) about Mint vs Ubuntu.

You may also want to use the 0.2 branch instead of master to a stable release.  Such as  
`./buildplus.sh Yourname youremail 0.2` if you are using the script or `git checkout 0.2` if building from source.

And of course, make sure to vote in the [poll](https://github.com/Motion-Project/motion/discussions/1794) in case you haven't already.
